src/main/webapp/app/modules/auth/complete-profile/complete-user-profile.tsx

```diff
@@ -0,0 +1,292 @@
+        </h2>
+        <Row className="justify-content-center">
+          <Col md="8">
+            {loading ? (
+              <p>Loading...</p>
+            ) : (
+              <AvForm model={userProfile} onSubmit={this.completeProfile}>
+                <AvGroup>
+                  <AvField
+                    id="first-name"
+                    label={translate('clubmanagementApp.userProfile.firstName')}
+                    type="text"
+                    name="firstName"
+                    validate={{
+                      required: { value: true, errorMessage: 'Please enter your First Name' },
+                      maxLength: { value: 200, errorMessage: 'First Name cannot be more than 200 characters' }
+                    }}
+                  />
+                </AvGroup>
+
+                <AvGroup>
+                  <AvField
+                    id="last-name"
+                    label={translate('clubmanagementApp.userProfile.lastName')}
+                    type="text"
+                    name="lastName"
+                    validate={{
+                      required: { value: true, errorMessage: 'Please enter your Last Name' },
+                      maxLength: { value: 200, errorMessage: 'Last Name cannot be more than 200 characters' }
+                    }}
+                  />
+                </AvGroup>
+
+                <AvGroup>
+                  <Label id="gender" for="gender">
+                    <Translate contentKey="clubmanagementApp.userProfile.gender">Gender</Translate>
+                  </Label>
+                  <AvInput
+                    id="gender"
+                    type="select"
+                    className="form-control"
+                    name="gender"
+                    validate={{
+                      required: { value: true, errorMessage: 'Please select your gender' }
+                    }}
+                  >
+                    <option value="" disabled>
+                      {translate('global.select.selectOne')}
+                    </option>
+                    <option value="FEMALE">{translate('clubmanagementApp.userProfile.genderValue.FEMALE')}</option>
+                    <option value="MALE">{translate('clubmanagementApp.userProfile.genderValue.MALE')}</option>
+                  </AvInput>
+                </AvGroup>
+
+                <AvGroup>
+                  <AvField
+                    id="phoneNumber"
+                    label={translate('clubmanagementApp.userProfile.phoneNumber')}
+                    type="text"
+                    name="phoneNumber"
+                    placeholder="E.g 123857221"
+                    validate={{
+                      required: { value: true, errorMessage: 'Please enter your Phone Number' },
+                      pattern: { value: '^([1-9][0-9]{6,20})$', errorMessage: 'Please enter a valid Phone Number without Country Code' }
```

Comment:
https://en.wikipedia.org/wiki/Telephone_numbers_in_Malaysia
malaysia phone number at most 10 digit but the regex allow up to 20 digits

Reply:
resolved in b24b1b151c6bb1e741153514eec75a336c4239fa